library/ssl_tls13_keys.c

```diff
@@ -820,4 +822,29 @@ int mbedtls_ssl_tls13_populate_transform( mbedtls_ssl_transform *transform,
     return( 0 );
 }
 
+int mbedtls_ssl_tls1_3_key_schedule_stage_early( mbedtls_ssl_context *ssl )
+{
+    int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
+    mbedtls_md_type_t md_type;
+    const unsigned char *input = NULL;
+    size_t input_len = 0;
+    if( ssl->handshake->ciphersuite_info == NULL )
```

Comment:
Please add a blank line just above to separate from the local variable declarations.